docs/docs/guides/how-to-parameterize-cassandra.md

```diff
@@ -0,0 +1,396 @@
+---
+title: How to set up an n-node Cassandra environment
+sidebar_label: Setting up an n-node Cassandra cluster
+slug: /how-to-parameterize-cassandra
+toc_max_heading_level: 2
+sidebar_position: 5
+---
+
+Introduction
+------------
+In this guide, we will set up a 3-node Cassandra cluster in Docker and parameterize the environment definition so it can easily be modified for use in different tests that require an _n_-node Cassandra cluster. Then we will show you how to run remotely hosted packages authored by others, and go through how to package and publish our work to Github for others to use as well.
+
+Specifically, we're going to configure our test environments with a way that allows us to both:
+1. Parameterize the environment so a user can trivially specify how many nodes they’d like for their system to have, and 
+2. Make their environment definition composable so that these environments can be easily included in tests with other services for different scenarios & use cases.
```

Comment:
```suggestion
2. Make our environment definition composable so that these environments can be included in tests with other services for different scenarios & use cases.
```